v22.1/take-full-and-incremental-backups.md

```diff
@@ -190,27 +194,27 @@ To define a specific subdirectory in the collection to restore from:
 
 <span class="version-tag">New in v22.1:</span> To explicitly control where your incremental backups go, use the [`incremental_location`](backup.html#options) option.
 
-In the following examples, the `{full_backup_location}` specifies the collection location containing the full backup. The `{incremental_backup_location}` is the alternative location that you can store an incremental backup:
+In the following examples, the `{collectionURI}` specifies the storage location containing the full backup. The `{explicit_incrementalsURI}` is the alternative location that you can store an incremental backup:
```

Comment:
is it worth adding an intro sentence to this subsection?:

 "By default, incremental backups are stored in the /incrementals subdirectory at the root of the collection. Advance users may want to store incremental backups in a different location."

Reply:
Although we have the advanced example section, I kind of shy away from defining users as "advanced".... What do you think about defining the use case instead? 

"By default, incremental backups are stored in the `/incrementals` subdirectory at the root of the collection. However, there are some advanced cases where you may want to store incremental backups in a different storage location."

I also have a potential follow-up project to build out a short section in the cloud storage docs re use cases.

Reply:
this looks good!